data/content/resources.md

```diff
@@ -0,0 +1,19 @@
+#### Learn
+
+- [A Brief Intro to Rust](https://ashleygwilliams.github.io/a-very-brief-intro-to-rust/#1)
+- The [Book](https://doc.rust-lang.org/book/)
+- The [Rust API Reference](https://doc.rust-lang.org/std/)
+- [Rust by Example](http://rustbyexample.com/)
+- The Rust [FAQ](https://www.rust-lang.org/en-US/faq.html)
+- [Rustlings](https://github.com/carols10cents/rustlings)
+- [Rust by Example](https://rustbyexample.com/)
+- [into_rust()](http://intorust.com/)
+- [exercism.io - Rust](http://exercism.io/languages/rust/about)
```

Comment:
As discussed, I only miss a link to https://community.rs/rustbridge/ here. We can think about embedding/migrating those materials later.